src/lib/config/index.ts

```diff
@@ -44,11 +44,22 @@ export const LOGGING = set(
 	}
 )
 
+export const POSTGRES = set(
+	process.env.POSTGRES || 'postgres://user:password@host:5433/lpg'
+)
+
 export const SESSION_SECRET = set(
 	process.env.SESSION_SECRET ||
 		'dcOVe-ZW3ul77l23GiQSNbTJtMRio87G2yUOUAk_otcbL3uywfyLMZ9NBmDMuuOt'
 )
 
+export const UI_SERVER = set('lpg.local.cshr.digital:3001', {
```

Comment:
These should be env variables configured on the host - we shouldn't need to change code to deploy to a new environment.